addon/current-user/service.js

```diff
@@ -110,6 +139,12 @@ export default Service.extend({
         );
       });
 
+      payload.composites.forEach((composite) => {
```

Comment:
`(payload.composites || []).forEach` to ensure we have an array to do the iterate on